.github/workflows/test-suite.yml

```diff
@@ -173,10 +156,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions-rs/toolchain@v1
+      - uses: helix-editor/rust-toolchain@v1
         with:
           profile: minimal
-          toolchain: nightly
+          toolchain: nightly-2024-06-25
```

Comment:
why do we fix nightly? the purpose of this test is to check our compatibility with coming version of meilisearch right. @Kerollmops who required for this test

Reply:
I think the purpose of this check is to check if the code is correctly formatted.

Because the stable version of `rustfmt` is very limited in options, we are using a nightly version.
I think tamo is right to be fixing it, to avoid that a regression on some nightly wrecks havoc in our CI (especially for formatting!)

Reply:
Maybe we should add to our "update Rust" template a reminder to also refresh the pinned Rust version, though.

Otherwise the nightly version will eventually become older than the stable version we're using, and that is probably not a great idea.

Reply:
> I think the purpose of this check is to check if the code is correctly formatted.

Yes, Louis is 100% right; we're using nightly options in rustfmt and thus need a nightly version of rustfmt, nothing else; we were not trying to catch any future bugs

> Maybe we should add to our "update Rust" template a reminder to also refresh the pinned Rust version, though.

Already done 😎

Reply:
Following you on this